contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -34,6 +34,19 @@
 
 #include "instr.h"
 
+/* We need to create a minimal stub of PLtsql_protocol_plugin so
+ * that we can reference pltsql_protocol_plugin_ptr to check whether
+ * we are coming from the TDS or PG client. This is required to support
+ * COPY inputs coming from DMS.
+ */
+typedef struct PLtsql_protocol_plugin
+{
+	/* True if Protocol being used by client is TDS. */
+	bool		is_tds_client;
+} PLtsql_protocol_plugin;
```

Comment:
I am still not sure how this is working, even in tds we are including the same plugins.
Check tds_int.h
```
#include "../../contrib/babelfishpg_tsql/src/pltsql.h"
#include "../../contrib/babelfishpg_tsql/src/pltsql-2.h"
```

Reply:
Updated to use a hook function. Note that this means that there is now an accompanying change in the engine code to define the hook.

Reply:
Reworked to use a rendezvous variable again.

Reply:
Removed this code, as a hook/rendezvous function is not needed anymore since we now just check a GUC.